tests/unit/client_tests.py

```diff
@@ -539,6 +539,38 @@ def test_constructor_with_account(self):
             'https://{0}.cloudant.com'.format(self.account)
             )
 
+    def test_bluemix_constructor(self):
+        """
+        Test instantiating a client object using a VCAP_SERVICES environment
+        variable.
+        """
+        instance_name = 'Cloudant NoSQL DB-lv'
+        vcap_services = {'cloudantNoSQLDB': [{
```

Comment:
Good thinking. I'll add an additional test. I'll also add a test for 1 service in VCAP without specifying `instance_name`.

`tests/unit/cloud_foundry_tests.py` is testing majority of this. But since I've already gaffed this once it'd be good to test again at the client instantiation level too.